app/controllers/notes_controller.rb

```diff
@@ -276,12 +277,20 @@ def search
 
   ##
   # Display a list of notes by a specified user
-  def mine
+  def user
     if params[:display_name]
       if @user = User.active.find_by(:display_name => params[:display_name])
         @params = params.permit(:display_name)
-        @title = t "notes.mine.title", :user => @user.display_name
-        @heading = t "notes.mine.heading", :user => @user.display_name
+
+        # set title/heading
+        if current_user && current_user.display_name == @user.display_name
+          @title = t "user.show.my notes"
+          @heading = t "user.show.my notes"
```

Comment:
I know that traces does something similar, but I wonder if this is sensible/worthwhile...

I mean `/mine` is basically a shortcut to asking for a particular user that isn't even preserved in the URL because we redirect, so giving this page a different title if you're logged in seems a little odd and not really necessary?

Reply:
I'm only copying what it does in the traces controller.. it basically just says "My Notes" if you're logged in instead of "Harry's Notes"

Reply:
Excerpt from traces controller:

```
@title = if target_user.nil?
               t ".public_traces"
             elsif current_user && current_user == target_user
               t ".my_traces"
```

Reply:
Just noticed that you referenced the traces page already, but for me I think its a worthwhile addition.
Throughout the whole site there are references to `My Profile` and `My Traces` and `My Settings` etc, so I'd say its only apt to apply the same to the Notes page!